cgi-bin/DW/Media.pm

```diff
@@ -110,9 +110,15 @@ sub upload_media {
 
     # set the security
     my $sec = $opts{security} || 'public';
+    $sec = 'usemask' if $sec =~ /^(?:friends|access)$/;
```

Comment:
Yeah, we're not good at following standards or best practices, especially as code has come from many sources over 15 years and many different reviewers. :)

You definitely don't need to change every other occurrence (unless you want to go on a code dive, which is always welcome!).